examples/BART/BART_introduction.ipynb

```diff
@@ -0,0 +1,576 @@
+{
```

Comment:
<p>the <code>(notebook_id)=</code>  is the identifier we'll then use from other notebooks to link here. It should be something specific to this notebook. I'd use the filename.</p><p> </p><p>The 2nd category should be like with tags, comma separated instead of a second category metadata key: <code>:category: intermediate, explanation</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/259/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>